packages/controllers/src/snaps/SnapController.test.ts

```diff
@@ -1998,51 +1880,45 @@ describe('SnapController', () => {
 
   describe('enableSnap/disableSnap', () => {
     it('updates snap state correctly', async () => {
-      const id = 'npm:example-snap';
-      const sourceCode = 'foo';
-      const manifest = getSnapManifest({
-        version: '1.0.0',
+      const manifest = {
+        ...FAKE_SNAP_MANIFEST,
         initialPermissions: { eth_accounts: {} },
-        shasum: getSnapSourceShasum(sourceCode),
-      });
+      };
 
       const snapController = getSnapController();
 
       await snapController.add({
         origin: FAKE_ORIGIN,
-        id,
+        id: FAKE_SNAP_ID,
         manifest,
-        sourceCode,
+        sourceCode: FAKE_SNAP_SOURCE_CODE,
       });
 
-      await snapController.disableSnap(id);
-      expect(snapController.get(id)?.enabled).toBe(false);
-      snapController.enableSnap(id);
-      expect(snapController.get(id)?.enabled).toBe(true);
+      await snapController.disableSnap(FAKE_SNAP_ID);
```

Comment:
Nit: When it's used many times in a row, would it make sense to have it defined in the test `const id = FAKE_SNAP_ID;`? For easy editing?